### PR TITLE
Shniak — code refactoring, change styles for my components, add transition of task list reference to TaskItem component

### PR DESCRIPTION
**Pull request by Warumo**
This pull request adds minor code refactor, add little change to few variable names in MainContainer, removes all inline styles so styling is now in separate SCSS files; adds task list reference transition from MainContainer component to TaskItem so Stas’ component can work correctly.

### CODE REVIEW

File: src/components/Dashboard/TasksColumns/TaskColumns.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  text-align: center;
+  font-size: 1.8rem;
+  color: rgb(118, 183, 192);
+  border-bottom: 3px solid rgb(116, 154, 159);
+}
+
+.taskColumnDoneTitle {
+  margin: 25px auto 1px;
+  text-align: center;
+  font-size: 1.8rem;
```
Comment (opanki) [inline src/components/Dashboard/TasksColumns/TaskColumns.scss]: please add additional class  `.taskColumn` for repeatable rules or group existing classes
`.taskColumnDoneTitle, .tasksColumnInProgressTitle, .tasksColumnToDoTitle { common rules here}`

File: src/components/Dashboard/MainContainer/MainContainer.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
+.mainContainer {
+  display: flex;
+  flex: 0 0 auto;
+  align-items: center;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+  min-height: calc(100vh - 100px);
+  min-width: 100vw;
+  background-color: rgb(247, 247, 247);
```
Comment (opanki) [inline src/components/Dashboard/MainContainer/MainContainer.scss]: it's more usable to keep color codes in variables

File: src/components/Dashboard/MainContainer/MainContainer.jsx [hunk 3]
```diff
@@ -12,30 +12,32 @@ export default class MainContainer extends Component {
...
-    const taskListRef = db.database().ref(`dashboards/${id}/taskList`);
+    const dashboardID = document.URL.split('/').pop();
+    const taskListRef = db.database().ref(`dashboards/${dashboardID}/taskList`);
     taskListRef.on('value', (snapshot) => {
       const taskListSnap = snapshot.val();
       const newState = [];
-      // eslint-disable-next-line no-restricted-syntax
-      for (const task in taskListSnap) {
+
+      Object.keys(taskListSnap).forEach(task => (
```
Comment (opanki) [inline src/components/Dashboard/MainContainer/MainContainer.jsx]: please move this logic to separate function and call it here
```
const newState = [];

      Object.keys(taskListSnap).forEach(task => (
        newState.push({
          id: task,
          name: taskListSnap[task].name,
          description: taskListSnap[task].description,
          status: taskListSnap[task].status,
          // subTaskList: taskListSnap[task].subTaskList,
          key: task,
        })));
```

File: src/components/Dashboard/TasksColumns/TaskColumns.scss (hunk 1 above)
Comment (Warumo) [reply src/components/Dashboard/TasksColumns/TaskColumns.scss]: Thank you for the review, I’ll correct everything as needed :)

**Pull request merged**